target/linux/bcm53xx/image/Makefile

```diff
@@ -135,25 +135,29 @@ define Device/asus
 endef
```

Comment:
Off-topic: Is this common definition unused or is it included somewhere don't see?

Reply:
I think it is unused.